lesson9.tex

```diff
@@ -0,0 +1,149 @@
+\documentclass{lessonplan}
+
+% Constants
+\newcommand{\lessonNumber}{9}
+\title{Lesson \lessonNumber: [Ciphers]}
+\author{\linkHome}
+\date{}
+
+% Buildup
+\setLessonNumber{\lessonNumber}
+
+
+%Document
+\begin{document}
+
+  \maketitle
+
+  \section{Overview}
+    \subsection{Summary}
+      Here we give an overview of the Lesson and what it teaches, to
+      both younger and older audiences
+
+      This lesson teaches students how to crack and create their own
+      ciphers through the context of an ancient treasure.
+
+      For more advanced students, this lesson goes into basic mathematics 
+      of prime number and modulo based ciphers. 
+
+    \subsection{Objectives}
+    \begin{itemize}
+      \item Introduce students to basic letter ciphers
+      \item (Intermediate) Introduce the concept of morse code and modern cyber security.
+      \item (Advanced) Describe the concept of prime number factorization for use in security systems.
+      \item (Cooperative) Have separate groups work together to help solve an ancient cipher!
+    \end{itemize}
+    \subsection{Key Vocabulary Terms}
+    \begin{itemize}
+      \item List some of the key takeaways from the lesson.  These are
+        also good things to ask about at the conclusion of the lesson
+      \item Cipher, Encryption, Decryption
+      \item (Intermediate) Cyber-security, Modulo
+      \item (Advanced) Prime-number factorization, RSA
+    \end{itemize}
+    \subsection{Materials}
+      \begin{itemize}
+        \item 2x Chromebooks for the group to work with for the Morse Code portion.
```

Comment:
Any sort of laptop will do, we should start making these more general. Also, is there a way to get the morse code content across without access to a laptop. Not necessary, but would be a nice +